lib/design_system/themes/custom_asset.dart

```diff
@@ -0,0 +1,7 @@
+enum CustomAsset {
+  loadingWoundCoAnimation('assets/lotties/wc_loading_animation.json');
```

Comment:
Wound CO animation?
We should use a project neutral animation.
I would say to just use a random royalty free lottie file from the web.

Reply:
Totally agree!! I just forgot to rename it